llvm/utils/repo/docker/README

```diff
@@ -0,0 +1,41 @@
+Ability to create a Docker image for 'pstore'
+
+Usage: build_docker_image.sh [options] [-- [cmake_args]...]
+
+Available options:
+  General:
+    -h|--help               show this help message
+  Docker-specific:
+    -t|--docker-tag         docker tag for the image
+  Checkout arguments:
+    -b|--branch             git branch to checkout, i.e. 'master',
+                            'branches/release_40'
+                            (default: 'master')
+    -l|--local              use local directory
+  Build-specific:
+    -i|--install-target     name of a cmake install target to build and
+                            include in the resulting archive. Can be
+                            specified multiple times.
+
+Required options: --docker-repository, at least one --install-target.
+
+All options after '--' are passed to CMake invocation.
+
+For example, running:
+  build_docker_image.sh \
+    --branch master --docker-tag latest \
+    --install-target install-clang --install-target install-pstore \
+    -- -DCMAKE_BUILD_TYPE=Release
+
+will produce a docker image:
+    pstore:latest - a small image with preinstalled release versions of
```

Comment:
As above the image should really be called "llvm-prepo" rather than pstore.

Reply:
Correct. Changed.